engine/api/templates.go

```diff
@@ -0,0 +1,72 @@
+package api
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/ovh/cds/engine/api/workflowtemplate"
+	"github.com/ovh/cds/engine/service"
+	"github.com/ovh/cds/sdk"
+)
+
+func (api *API) getTemplatesHandler() service.Handler {
+	return func(ctx context.Context, w http.ResponseWriter, r *http.Request) error {
+		ts, err := workflowtemplate.GetAll(api.mustDB())
+		if err != nil {
+			return err
+		}
+
+		return service.WriteJSON(w, ts, http.StatusOK)
+	}
+}
+
+func (api *API) postTemplateHandler() service.Handler {
+	return func(ctx context.Context, w http.ResponseWriter, r *http.Request) error {
+		t := &sdk.WorkflowTemplate{}
```

Comment:
Proposal: Not very fan of this writing, I prefer `t := sdk.WorkflowTemplate{}` or `var t sdk.WorkflowTemplate`. Why ? because in this case it's explicit when in functions below you indicate `UnmarshalBody(r, &t)` when we see &whatEver we now that we SHARE the value. (cf: https://www.youtube.com/watch?v=2557w0qsDV0)